composer.json

```diff
@@ -29,7 +28,18 @@
     "symfony/config": "^2.8",
     "symfony/translation": "^2.8",
     "symfony/yaml": "~2.7",
-    "vlucas/phpdotenv": "^2.4"
+    "vlucas/phpdotenv": "^2.4",
+
```

Comment:
just a heads up, whitespaces in composer.json will get smashed by auto-rewrite tools (cow is one of them) :)